samples/net/sockets/http_server/src/main.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023, Emna Rekik
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/net/http/server.h>
+#include <zephyr/posix/netinet/in.h>
```

Comment:
For apps (samples and tests), this can be `<netinet/in.h>` if you have `CONFIG_POSIX_API=y` in your `prj.conf`.

For Zephyr library code, we should use the `<zephyr/posix/...>` prefix though.

Reply:
Also, the "canonical" way that we should include headers is now in 3 blocks.

It's kind of unofficial right now, but better to be future-proof.

Each block is ordered alphabetically / lexically

```cpp
/* private headers */
#include "private_header.h"

/* standard headers (e.g. ISO C, POSIX) */
#include <errno.h>
#include <unistd.h>

/* zephyr headers */
#include <zephyr/kernel.h>
#include <zephyr/sys/bitarray.h>
```